### ISSUE TITLE
In FileSets Thumbnail Image Link missing a text alternative

### ISSUE THREAD

**Issue by posixeleni** [samvera/hyku#3047]
vpat.hykudemo.org
Version 7.1

### Descriptive summary

Add a text alternative for thumbnail image link in File Set for screen reader to read out what clicking on this link will do. Right now clicking on the link on the thumbnail does nothing, which is confusing for any user.

Links represented by images or icons can be problematic for visitors who are blind or have low vision.

The text alternative should describe the link's purpose — or what visitors can expect to find after they click it.

Issue is found in HTML:
`<h2 class="sr-only">Downloadable Content</h2>
      <a href="[https://vpat.hykudemo.org/concern/file_sets/5584dabf-bb84-4b9e-a75d-3af4d6798ec9](view-source:https://vpat.hykudemo.org/concern/file_sets/5584dabf-bb84-4b9e-a75d-3af4d6798ec9)"><img class="representative-media" alt="Explanation of what a VPAT is" role="presentation" src="[/downloads/5584dabf-bb84-4b9e-a75d-3af4d6798ec9?file=thumbnail](view-source:https://vpat.hykudemo.org/downloads/5584dabf-bb84-4b9e-a75d-3af4d6798ec9?file=thumbnail)" /></a>`

See also: 
    WCAG 2.2 A [2.4.4: Link Purpose in Context](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context)
    WCAG 2.2 AAA [2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-link-only)
    WCAG 2.2 A [4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value)

### Rationale
Accessibility issue remediation

### Expected behavior

Either remove the <a> tag entirely since link goes nowhere when clicked or one solution that is recommended by SiteImprove and WCAG https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8 is to add an aria-label in the <a> tag that explains what clicking on the image will result in. For example:

`<a href-="..." aria-label="Download/View image">`

### Actual behavior

Right now clicking on the link does nothing and there's no alternative text explaining what the link will do.

### Steps to reproduce the behavior
Go to https://vpat.hykudemo.org/concern/file_sets/5584dabf-bb84-4b9e-a75d-3af4d6798ec9?locale=en
Click on SiteImprove Plugin and see issue shown in screenshot
<img width="910" height="996" alt="Image" src="https://github.com/user-attachments/assets/eb9d76b8-a7c9-42e7-8729-e03b0be99473" />
Issue is found in HTML:
`<h2 class="sr-only">Downloadable Content</h2>
      <a href="[https://vpat.hykudemo.org/concern/file_sets/5584dabf-bb84-4b9e-a75d-3af4d6798ec9](view-source:https://vpat.hykudemo.org/concern/file_sets/5584dabf-bb84-4b9e-a75d-3af4d6798ec9)"><img class="representative-media" alt="Explanation of what a VPAT is" role="presentation" src="[/downloads/5584dabf-bb84-4b9e-a75d-3af4d6798ec9?file=thumbnail](view-source:https://vpat.hykudemo.org/downloads/5584dabf-bb84-4b9e-a75d-3af4d6798ec9?file=thumbnail)" /></a>`

**Issue can be found on these pages:**

https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=de
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=en
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=es
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=fr
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=it
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=pt-BR
https://vpat.hykudemo.org/concern/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=zh
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=de
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=en
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=es
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=fr
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=it
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=pt-BR
https://vpat.hykudemo.org/concern/parent/5889acd4-175f-42f4-ae5e-31f3dcf7d54b/file_sets/c9bd9ecf-1872-4b53-9675-a4496c5067f5?locale=zh
